pom.xml

```diff
@@ -139,7 +139,10 @@
               <excludes>
                 <exclude>@org.apache.hadoop.classification.InterfaceStability$Evolving</exclude>
                 <exclude>@org.apache.hadoop.classification.InterfaceStability$Unstable</exclude>
-                <exclude>com.datatorrent.contrib.parquet</exclude>
+                <exclude>com.datatorrent.lib</exclude>
```

Comment:
Are those excludes required if the major version is changed?

Reply:
No, they won't be needed. Should I make the version change in the same commit?

Reply:
My take - yes. Is there a reason not to?